_posts/2019-06-25-lambda-quickstart.md

```diff
@@ -0,0 +1,320 @@
+---
+title: How to configure OpenFaaS functions to execute on AWS Lambda
+description: In a follow up article Edward Wilde shows how to configure OpenFaaS functions to execute on AWS Lamda using the new faas-lambda provider.
+date: 2019-06-25
+image: /images/faas-quickstart/backdrop.jpg
+categories:
+  - lambda
+  - automation
+  - tutorial
+author_staff_member: ed
+dark_background: true
+---
+
+In this post we are going to follow the steps necessary to configure OpenFaaS to execute function on AWS Lambda using 
+the new faas-lambda provider. See the previous blog post [announcing support for AWS Lambda] for an overview of the 
+technology.
+
+
+### Pre-requisites
```

Comment:
Rather than installing full EKS, I would suggest adding instructions for bootstrapping a 1-node VM on EC2 with k3s.

Reply:
The cost would be tiny in comparison, I think the cost could put people off. Let me know if you need help with that.

Reply:
I don't have any experience using k3s. I only want to include a pre-requsite that has been fully tested with the steps in this article. If you have time to do that with k3s, then it sounds like a good ideas. Otherwise suggest we leave as it for the time being